crm_phonecall_plan/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+      :alt: License: AGPL-3
+
+==============
+CRM Phonecall Plan
+==============
+
+This module adds a wizard to allow you to plan calls from the lead and partner views.
+
+Usage
+=====
+
+#. Go to partners/leads view
+#. Pick partners/leads using checkboxes
+#. Click on "Plan a call" in the more submenu
+#. Fill in the wizard with information you need and validate.
+
+   .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+      :alt: Try me on Runbot
+         :target: https://runbot.odoo-community.org/runbot/111/8.0
```

Comment:
Hi @misern2. I have seen that the link points to the top-menu entry: website. If you are entering for first time you can change the url with http://3293144-94-bbe389.runbot2.odoo-community.org/web to enter the login page.I hope this helps.